src/routes/landing.js

```diff
@@ -21,6 +21,13 @@ class Landing extends React.Component {
             />
           </FullWidthContainer>
 
+          <FullWidthContainer flex={columnContainerFlex}>
+            <Button
+              title='Skip search and go to results'
```

Comment:
lol, this makes no sense in a final version but fine for now. Actually, a comment saying this is temporary might be nice